src/main/resources/com/puppycrawl/tools/checkstyle/meta/checks/regexp/RegexpMultilineCheck.xml

```diff
@@ -3,7 +3,7 @@
    <module>
       <check fully-qualified-name="com.puppycrawl.tools.checkstyle.checks.regexp.RegexpMultilineCheck"
              name="RegexpMultiline"
-             parent="com.puppycrawl.tools.checkstyle.Checker">
+             parent="">
```

Comment:
Please revert this change. It is showing parent module to use when using cli.

Reply:
and this same is causing the check failures

Reply:
Yes, i didn't made change to this file. Idk how it got changed. But have changed it back now.